src/statusBar.ts

```diff
@@ -55,6 +55,17 @@ class StatusBarImpl implements vscode.Disposable {
     this._statusBarItem.dispose();
   }
 
+  public Get() {
```

Comment:
:( why on earth did we do pascal case for this file? what idiot did that (fyi. it was me).